src/routes/servers.js

```diff
@@ -3,10 +3,12 @@ const { StatusCodes } = require('http-status-codes');
 
 const express = require('express');
 const router = express.Router();
+const servers = require('../storage/servers');
 
 const db = require('../storage/DataAccess');
 
 const COLLECTION = 'servers';
```

Comment:
Wont let me add a comment on line 16 so I'll do it here... might as well replace the get all servers with a method in the other servers.js file. Perhaps rename `getServer(serverId)` to `getServerById(serverId)` and then implement `getServers()` to return all.